QuestionBot/QuestionBot/Model/MessageEmitter.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace QuestionBot.Model {
+    public class MessageEmitter : IMessageEmitter {
+        private string _lineInput = "";
+        private static IList<IMessageListener> _listeners = new List<IMessageListener>();
```

Comment:
I think I like _listeners as an instance member better - it just seems more versatile if we ever want to use more MessageEmitters